popUp.js

```diff
@@ -0,0 +1,170 @@
+const projects = [
+  {
+    title: 'TONIC 1',
+    prof: ['Canopy', 'back end Dev', '2015'],
+    src: 'images/SnapshootPortfolio.png',
+    description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
+    technologies: ['html', 'css', 'javascript', 'Github', 'Ruby', 'Bootstrap'],
+    seeLiveLink: '#',
+    seeSourceLink: '#',
+  },
+  {
+    title: 'TONIC 2',
+
+    prof: ['Canopy', 'back end Dev', '2015'],
+    src: 'images/SnapshootPortfolio.png',
+    description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe."],
```

Comment:
- [Optional] Please fix the button on the desktop view, it is too stretched as compared to the figma design, so that your portfolio looks professional.

##### Your design
![Screenshot from 2022-06-18 16-35-15](https://user-images.githubusercontent.com/58771507/174445766-a0564c1f-a178-401a-af6b-4a4a7cf194ca.png)